requirements.txt

```diff
@@ -13,3 +13,4 @@ ipykernel
 segmentation_models
 tqdm
 janus==0.6.1
+folium==0.2.1
```

Comment:
What is this for?

Reply:
it seems segmentation-model use a version of datascience==0.6.0, which requires folium==0.2.1. google colab gives errors if not, but even with error, it can still works. just weird